shared/gast_to_code/general_helpers.py

```diff
@@ -60,5 +60,7 @@ def java_linter(output_code):
     output_code = output_code.replace("{;", "{")
     output_code = output_code.replace("};", "}")
     output_code = output_code.replace("+;", "+")
+    output_code = output_code.replace("\t;", "\t")
+    output_code = output_code.replace(".;", ".")
```

Comment:
Ha for some funny breaking cases, console.log() with emojis would probably break some of these cases.

However... Given the time constraints I think this is okay for now. But this should definitely have a TODO(<taiga-ticket-#>)